docs/examples/google-core-iot/README.md

```diff
@@ -35,9 +35,10 @@ The first thing we need to do is to download our the certificate of the device.
 
 ```
 $ export DEVICE_ID=mydevice.d.wott.local
-$ curl -s "https://api.wott.io/v0.2/device-cert/$DEVICE_ID" | jq -r '.certificate' > device.crt
+$ curl -s "https://api.wott.io/v0.2/device-cert/$DEVICE_ID"
```

Comment:
Don't forget the redirection (`> device.crt`) otherwise we don't get the cert required in the next step.

Reply:
fixed